business-wiki-frontend/pages/_app.tsx

```diff
@@ -2,10 +2,10 @@ import "../pages/styles.scss";
 import { AppProps } from 'next/app';
 import { useState } from 'react';
 import { UserContext } from '../shared/contexts/UserContext';
-import { User } from '../shared/models/User';
+import { UserInterface } from '../shared/models/UserInterface';
 
 export default function MyApp({ Component, pageProps }: AppProps) {
-    const [user, setUser] = useState({} as User);
+    const [user, setUser] = useState({} as UserInterface);
 
     return (
         <UserContext.Provider value={
```

Comment:
ES6, jeśli masz taką sytuacje, że user:user, to możesz zapisać skrótowo jako user

Reply:
Resolved in cf206164ec51e717f847936ec8e7708f26e75f66